meta-lxatac-software/recipes-devtools/python/python3-labgrid/environment

```diff
@@ -1,4 +1,8 @@
+# Changes to this file will be migrated during rauc installs.
+# Feel free to change this configuration if needed.
+
 # This is a 'portable' labgrid configuration.
 # Make sure that a host 'labgrid' can be resolved using your DNS searchlist.
+# Otherwise change it to your needds.
 LABGRID_COORDINATOR_IP=labgrid
```

Comment:
I would prefer to have an empty line between the comment and the content for esthetic reasons.
But I could also live without it.

Reply:
I don't have an opinion on that detail. I've added the empty line.